apps/tyler-site/app/[slug]/page.tsx

```
@@ -1,19 +1,43 @@
-  // This is a placeholder - will be replaced when React Bricks is configured
   return (
-    <main className="min-h-screen flex items-center justify-center">
-      <div className="text-center">
-        <h1 className="text-4xl font-bold mb-4">Page: {slug}</h1>
-        <p className="text-gray-600">
-          Configure React Bricks to see dynamic content here
-        </p>
-      </div>
-    </main>
+    <>
+      <div dangerouslySetInnerHTML={{ __html: page.html }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)